integration-tests/external-plugin-otel-test/external-plugin-otel-test-app/pom.xml

```diff
@@ -44,10 +44,20 @@
     <build>
         <finalName>${project.artifactId}</finalName>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
[nit] default maven plugin `groupId` can be omitted.

Is there any particular reason the jar plugin is required here ? For example to run it in the IDE ?
The packaged version is provided with dependencies with `maven-assembly-plugin` below.

Reply:
The integration tests have been adapted to use two versions of this app: One containing the Otel-dependencies (`jar-with-dependencies`) and one without (the "standard" jar built by the maven-jar-plugin).

The entry for the `maven-jar-plugin` is required to add the `Main-Class` entry to the Jar without the dependencies.